### DISCUSSION TITLE
Bookmark: start a set of Bookmark

### DISCUSSION THREAD

**Discussion by BambuschJ** [dnGrep/dnGrep#582] [Ideas]
Hello doug24,
I use dnGrep to convert SAP-export lists.
For this it is necessary to do several regex expressions in a certain order.
Here is my suggestion:
     add additional fields for Set and Set-Position in the bookmark window. A function for playing a set via the menu and the command line.

Additional suggestion for the bookmark window:
     Definition of the field order by the user
     Possibility to sort bookmarks
	 
in the attachment I have put together an example -- In advance many thanks

[dnGrep_Example4SAPLists.zip](https://github.com/dnGrep/dnGrep/files/7448159/dnGrep_Example4SAPLists.zip)
Best regards

J. Bambusch

**Comment by doug24** [comment]
@BambuschJ 
Somehow, I missed or forgot about this.  I looked at your example and I will think about it - it sounds like a good idea.
